css/css-animations/WEB_FEATURES.yml

```diff
@@ -18,6 +18,9 @@ features:
 - name: empty
   files:
   - empty-pseudo-class-with-animation.html
+- name: hsl
+  files:
+  - animate-with-background-color-hsla-to-legacy-rgb.html
```

Comment:
Hmm, this test also uses relative color syntax, which is a much newer feature than `hsl()` and `hsla()`. I think just leave it with animations-css.

Reply:
HSL still feels like experimental technology to me. I guess I'm much older than I feel :(

Reply:
Forever young!